Makefile

```diff
@@ -5,6 +5,13 @@ UNAME:= $(shell uname -o)
 PROJECT_ROOT := $(shell git rev-parse --show-toplevel)
 CONTAINERS_RUNNING := $(shell docker ps -q --filter "name=lumigator-")
 
+-include .env
+# GPU related settings
+#
+RAY_WORKER_GPUS ?= "0.0"
+RAY_WORKER_GPUS_FRACTION ?= "0.0"
+INFERENCE_PIP_REQS ?= ../jobs/inference/requirements_cpu.txt
+
```

Comment:
I think including the .env here is perfect , but, shouldn't we put this vars that you just added in the .env as well? you can put defaults in the .env.example file

Reply:
How would this affect cloud/hybrid settings? On those, the user should be able to use a GPU.